### PR TITLE
Add original asset names for all object_d* files

### PR DESCRIPTION
**Pull request by tom-overton**
I discovered that we can actually get original asset names for most `CollisionHeaders` by using a different tool for viewing MM3D assets than I normally use; I went back and added names for collision in `object_a*` through `object_c*` files.

I also noticed that in `object_dora`, two of the DLs were misnamed, so I just swapped them to fix it.

Apologies for not doing much translation on this one.

### CODE REVIEW

File: assets/xml/objects/object_am.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Root>
     <File Name="object_am" Segment="6">
-        <Collision Name="object_am_Colheader_000118" Offset="0x118" />
+        <Collision Name="object_am_Colheader_000118" Offset="0x118" /> <!-- Original name is "am_bgdatainfo" -->
```
Comment (isghj5) [inline assets/xml/objects/object_am.xml:3]: This is "am_" but the collider is "armos_" ? Guess that supports the theory armos was thought of second

File: assets/xml/objects/object_dai.xml [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
+        <DList Name="object_dai_DL_0002E8" Offset="0x2E8" /> <!-- Original name is "dai_ball_modelT" -->
+        <Animation Name="object_dai_Anim_000CEC" Offset="0xCEC" /> <!-- Original name is "dg_haku" ("to breathe out​") -->
+        <Animation Name="object_dai_Anim_0010F8" Offset="0x10F8" /> <!-- Original name is "dg_iare" ("are" = "huh?; eh?"") -->
+        <Animation Name="object_dai_Anim_0014BC" Offset="0x14BC" /> <!-- Original name is "dg_iareend" -->
+        <Animation Name="object_dai_Anim_001E44" Offset="0x1E44" /> <!-- Original name is "dg_iareloop" -->
+        <Animation Name="object_dai_Anim_002E58" Offset="0x2E58" /> <!-- Original name is "dg_inemuri" ("nemuri" = "to fall asleep") -->
+        <Animation Name="object_dai_Anim_003CAC" Offset="0x3CAC" /> <!-- Original name is "dg_isuwaru" ("suwari" = "to sit" )-->
+        <Animation Name="object_dai_Anim_0040E0" Offset="0x40E0" /> <!-- Original name is "dg_isuwaruloop" -->
+        <Animation Name="object_dai_Anim_0048B4" Offset="0x48B4" /> <!-- Original name is "dg_iwait" -->
+        <Animation Name="object_dai_Anim_005100" Offset="0x5100" /> <!-- Original name is "dg_iwalk" -->
```
Comment (isghj5) [inline assets/xml/objects/object_dai.xml:14]: Huh, he _does_ have a walking animation... is that left over from OOT? (he never walks, just sits in snowhead..)

File: assets/xml/objects/object_daiku.xml [hunk 3]
```diff
@@ -66,11 +66,11 @@
...
-        <Animation Name="object_daiku_Anim_00B690" Offset="0xB690" />
-        <Animation Name="object_daiku_Anim_00BEAC" Offset="0xBEAC" />
-        <Animation Name="object_daiku_Anim_00C234" Offset="0xC234" />
-        <Animation Name="object_daiku_Anim_00C92C" Offset="0xC92C" />
-        <Animation Name="object_daiku_Anim_00D328" Offset="0xD328" />
+        <Animation Name="object_daiku_Anim_00ACD0" Offset="0xACD0" /> <!-- Original name is "dk2_roya" -->
+        <Animation Name="object_daiku_Anim_00B690" Offset="0xB690" /> <!-- Original name is "dk2_shiji" (perhaps "instructions; directions​"?) -->
+        <Animation Name="object_daiku_Anim_00BEAC" Offset="0xBEAC" /> <!-- Original name is "dk2_shiji02" -->
+        <Animation Name="object_daiku_Anim_00C234" Offset="0xC234" /> <!-- Original name is "dk2_suwari" ("sitting") -->
+        <Animation Name="object_daiku_Anim_00C92C" Offset="0xC92C" /> <!-- Original name is "dk2_tasuke" ("to save; to rescue​") -->
```
Comment (isghj5) [inline assets/xml/objects/object_daiku.xml:73]: Oh neat this animation survived from OOT, huh.

File: assets/xml/objects/object_delf.xml [hunk 4]
```diff
@@ -1,82 +1,82 @@
...
+        <Animation Name="object_delf_Anim_00CEE0" Offset="0xCEE0" /> <!-- Original name is "elf_nanikairu2" -->
+        <Animation Name="object_delf_Anim_00D260" Offset="0xD260" /> <!-- Original name is "elf_nanikairu_loop2" -->
+        <Animation Name="object_delf_Anim_00E024" Offset="0xE024" /> <!-- Original name is "elf_okizari" -->
+        <Animation Name="object_delf_Anim_00EB18" Offset="0xEB18" /> <!-- Original name is "elf_okizari_no" -->
+        <Animation Name="object_delf_Anim_00EBB0" Offset="0xEBB0" /> <!-- Original name is "elf_okizari_sakebu" -->
+        <Animation Name="object_delf_Anim_00EE30" Offset="0xEE30" /> <!-- Original name is "elf_okizari_sakebu_wait" -->
+        <Animation Name="object_delf_Anim_00F0D0" Offset="0xF0D0" /> <!-- Original name is "elf_okizari_wait" -->
+        <Animation Name="object_delf_Anim_00FB30" Offset="0xFB30" /> <!-- Original name is "elf_sakimawari_1" -->
+        <Animation Name="object_delf_Anim_010590" Offset="0x10590" /> <!-- Original name is "elf_sakimawari_2" -->
+        <Animation Name="object_delf_Anim_01088C" Offset="0x1088C" /> <!-- Original name is "elf_samui" -->
```
Comment (isghj5) [inline assets/xml/objects/object_delf.xml:63]: Hmm, so the cutscene guy likes his onomatopoeia over engrish..

File: assets/xml/objects/object_dnt.xml [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
+        <Animation Name="object_dnt_Anim_0038CC" Offset="0x38CC" /> <!-- Original name is "dnt_onedari" -->
+        <Animation Name="object_dnt_Anim_003CC0" Offset="0x3CC0" /> <!-- Original name is "dnt_onedariTOwait" -->
         <Blob Name="object_dnt_Blob_003CD0" Size="0x10" Offset="0x3CD0" />
-        <Animation Name="object_dnt_Anim_004700" Offset="0x4700" />
-        <Animation Name="object_dnt_Anim_004AA0" Offset="0x4AA0" />
-        <Animation Name="object_dnt_Anim_004E38" Offset="0x4E38" />
-        <Animation Name="object_dnt_Anim_005488" Offset="0x5488" />
-        <Animation Name="object_dnt_Anim_00577C" Offset="0x577C" />
-        <Animation Name="object_dnt_Anim_005CA8" Offset="0x5CA8" />
+        <Animation Name="object_dnt_Anim_004700" Offset="0x4700" /> <!-- Original name is "dnt_spoon" -->
```
Comment (isghj5) [inline assets/xml/objects/object_dnt.xml:17]: wat

File: assets/xml/objects/object_driftice.xml [hunk 6]
```diff
@@ -5,7 +5,7 @@
         <Texture Name="object_driftice_TLUT_000420" OutName="tlut_000420" Format="rgba16" Width="4" Height="4" Offset="0x420" />
         <Texture Name="object_driftice_Tex_000440" OutName="tex_000440" Format="ci4" Width="64" Height="64" Offset="0x440" />
         <Texture Name="object_driftice_Tex_000C40" OutName="tex_000C40" Format="ci4" Width="64" Height="64" Offset="0xC40" />
-        <DList Name="object_driftice_DL_0016A0" Offset="0x16A0" />
-        <Collision Name="object_driftice_Colheader_001AA8" Offset="0x1AA8" />
+        <DList Name="object_driftice_DL_0016A0" Offset="0x16A0" /> <!-- Original name is "obj_iceblock_model" -->
+        <Collision Name="object_driftice_Colheader_001AA8" Offset="0x1AA8" /> <!-- Original name is "obj_iceblock_bgdatainfo" -->
```
Comment (isghj5) [inline assets/xml/objects/object_driftice.xml:9]: Wait are you sure? there is an actual `obj_iceblock` object which is separate, is this a duplicate?

**Review by isghj5 — APPROVED** [review]

File: assets/xml/objects/object_driftice.xml (hunk 6 above)
Comment (tom-overton) [reply assets/xml/objects/object_driftice.xml:9]: Yes, I'm sure. `object_ice_block` still exists in MM3D, and has assets like this:
image

File: assets/xml/objects/object_daiku.xml (hunk 3 above)
Comment (tom-overton) [reply assets/xml/objects/object_daiku.xml:73]: Looking at EnDaiku, it looks like there's code to play this animation? I have no idea under what conditions it is triggered though.

File: assets/xml/objects/object_am.xml (hunk 1 above)
Comment (tom-overton) [reply assets/xml/objects/object_am.xml:3]: Yeah, it's really weird. I had to double-check this one to make absolutely sure I was labeling the two of them correctly.

File: assets/xml/objects/object_dai.xml (hunk 2 above)
Comment (tom-overton) [reply assets/xml/objects/object_dai.xml:14]: This object is also used for the Goron with the same name as the player, who walks around. I don't know why these two share an object, but they do.

File: assets/xml/objects/object_dnt.xml (hunk 5 above)
Comment (tom-overton) [reply assets/xml/objects/object_dnt.xml:17]: Yeah it looks like that's the correct animation. I have absolutely zero idea why it is called "spoon"
spoon

File: assets/xml/objects/object_dai.xml (hunk 2 above)
Comment (isghj5) [reply assets/xml/objects/object_dai.xml:14]: Wait, goron with the same name as the player? doesnt he have a hat? was that a DL I missed?

File: assets/xml/objects/object_daiku.xml (hunk 3 above)
Comment (isghj5) [reply assets/xml/objects/object_daiku.xml:73]: Uhhh... in hindsight maybe the credits shows them sitting down?

File: assets/xml/objects/object_dai.xml (hunk 2 above)
Comment (tom-overton) [reply assets/xml/objects/object_dai.xml:14]: Yeah, there are a few of DLs in there for that Goron's clothes. In MM3D, they're all just part of the skeletal mesh:
image

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/objects/object_dora.xml [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
+        <DList Name="gKendoKanbanTopRightDL" Offset="0x2080" /> <!-- Original name is "ita_tR_model" -->
+        <DList Name="gKendoKanbanTopLeftDL" Offset="0x2180" /> <!-- Original name is "ita_tL_model" -->
+        <DList Name="gKendoKanbanBottomLeftDL" Offset="0x2280" /> <!-- Original name is "ita_bL_model" -->
+        <DList Name="gKendoKanbanBottomRightDL" Offset="0x2380" /> <!-- Original name is "ita_bR_model" -->
         <Texture Name="gKendoKanbanSplitFrontTex" OutName="kendo_kanban_front_split_alt" Format="rgba16" Width="32" Height="64" Offset="0x2480" />
         <Texture Name="gKendoKanbanSplitSideTex" OutName="kendo_kanban_side_split_alt" Format="rgba16" Width="32" Height="32" Offset="0x3480" />
-        <DList Name="gDoraChainDL" Offset="0x3FD0" />
-        <DList Name="gDoraGongDL" Offset="0x4160" />
+        <DList Name="gDoraGongDL" Offset="0x3FD0" /> <!-- Original name is "dora_hontai_model" -->
+        <DList Name="gDoraChainDL" Offset="0x4160" /> <!-- Original name is "kusariT_model" -->
```
Comment (engineer124) [inline assets/xml/objects/object_dora.xml:14]: So these were swapped?

**Review by engineer124 — APPROVED** [review]

**Pull request merged**